examples/mkcert/devenv.nix

```diff
@@ -0,0 +1,20 @@
+{ pkgs, config, ... }:
+
+{
+  mkcert.domains = [
+    "example.com"
```

Comment:
This interface is like easy, but not flexible. What if I want to have one certificate with `*.example.com` and `example.com`. 

Also the referencing directly into the state feels not like a good ux 🙈. Maybe a helper to assemble the path 🤔

Reply:
Do you have an example on how to add such a helper for the path? 🤔